setup.py

```diff
@@ -44,7 +44,7 @@
         author=__author__,
         license=__license__,
         description=__doc__,
-        build_number=int(getpass("build number: ")),
+        build_number=getpass("build number: "),
```

Comment:
Now that they can be strings, having a convention for build numbers would be helpful.

Reply:
I used <build-#-at-start-of-branch>-<pr-github-id>-<counter>. On main it has been an incrementing counter, not sure how this will be resolved when merging branches as each can increment on their own. Defer this decision to a later time